WcaOnRails/app/views/competitions/show_events.html.erb

```diff
@@ -0,0 +1,68 @@
+<% provide(:title, @competition.name) %>
```

Comment:
I have a general comment/question on this page: its aim is to show a summary of all the cutoff/time limit requirements for the competition?
Does the organizer/delegate have a way to change the data in there?